upf/upf_pfcp_api.c

```diff
@@ -1801,6 +1803,12 @@ handle_create_urr (upf_session_t * sx, pfcp_create_urr_t * create_urr,
 	create->time_quota.period = urr->time_quota;
 	create->time_quota.base = now;
       }
+    if (ISSET_BIT (urr->grp.fields, CREATE_URR_QUOTA_VALIDITY_TIME))
+      {
+	create->update_flags |= PFCP_URR_UPDATE_QUOTA_VALIDITY_TIME;
+	create->quota_validity_time.period = urr->quota_validity_time;
+	create->quota_validity_time.base = now;
+      }
```

Comment:
wrong order, Quota Validity Time should go after Dropped DL
Traffic Threshold  and before Monitoring Time to keep it in sync with TS 29.244.